### PR TITLE
Dc/bug/save state upon refresh

### PR DESCRIPTION
**Pull request by orangeboy55555**
Fix the issue where the user gets logged out every time they hit refresh

### CODE REVIEW

File: frontend/src/components/home/Home.jsx [hunk 1]
```diff
@@ -1,30 +1,20 @@
...
   const navigate = useNavigate();
+  const { userId } = JSON.parse(sessionStorage.getItem("session"));
 
   function handleHomeClick() {
     navigate("/");
   }
 
   function handleLogout() {
     dispatch({ type: "CLEAR", payload: "" });
+    sessionStorage.setItem("session", null);
```
Comment (ShahzaibParacha) [inline frontend/src/components/home/Home.jsx:17]: This should be `sessionStorage.clear()`, setting it to null means the key is still there but not associated with a `null` value
https://developer.mozilla.org/en-US/docs/Web/API/Storage/clear

**Review by ShahzaibParacha — APPROVED** [review]
All good, but that one line of code needs to be changed. Can do it in one of the later PRs.

**Review by ShahzaibParacha — APPROVED** [review]
This clears the issue

**Review by DaTanUmanitoba — APPROVED** [review]
It looks all good.

**Pull request merged**